internal/xds/translator/testdata/in/xds-ir/jsonpatch.yaml

```diff
@@ -21,15 +21,15 @@ jsonPatches:
   operation:
     op: "add"
     path: "/virtual_hosts/0/rate_limits"
-    value:
-    - actions:
-      - remote_address: {}
+    value: |
+      - actions:
+        - remote_address: {}
 - type: "type.googleapis.com/envoy.config.cluster.v3.Cluster"
   name: rate-limit-cluster
   operation:
     op: add
     path: ""
-    value:
+    value: |
```

Comment:
is the '|' required? if so, I don't think this is better than before.

Reply:
The  "|" means that the value is a multi-line string, that's it.

Reply:
It's ok for a configmap.
I think here we want some like `pb.Struct` or `map[string]interface{}`, so I prefer `apiext.JSON` than `string`.

Reply:
The value of the Patch is a string. Besides, it's very hard to troubleshooting if we define the API as a byte array.

Reply:
I does look a bit weird. It's better to just treat the value as a yaml/json structure.